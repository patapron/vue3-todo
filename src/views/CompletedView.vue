<template>
  <TodoList :name2="'Completed'" :filterValue="true" />
</template>

<script>
import { defineComponent } from 'vue';

// Components
import TodoList from '../components/TodoList.vue';

export default defineComponent({
  name: 'CompletedView',

  components: {
    TodoList,
  },
});
</script>
