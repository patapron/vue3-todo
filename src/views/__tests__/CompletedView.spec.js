/* eslint-disable prettier/prettier */
import  CompletedView from "@/views/CompletedView.vue";
import { shallowMount } from "@vue/test-utils";

// import { MODULE_CODE } from '../../../config/';

describe("CompletedView", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(CompletedView, {
      mocks: {
        $t: () => "Test text",
        $can: () => true,
      },
    });
  });

  it("renders correctly", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  //   it('module name', () => {
  //     expect(wrapper.vm.moduleName).toEqual(MODULE_CODE);
  //   });
});
