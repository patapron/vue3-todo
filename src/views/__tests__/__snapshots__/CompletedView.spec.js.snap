// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompletedView renders correctly 1`] = `<todo-list-stub name2="Completed" filtervalue="true"></todo-list-stub>`;
